src/models/Fruit.ts

```
@@ -0,0 +1,25 @@
+export const FruitSchema = new Schema<IFruit>({
+  id: { type: Schema.Types.ObjectId },
+  name: { type: String, index: { unique: true } },
+  description: { type: String }
+});
+
+FruitSchema.set('toJSON', {
+  virtuals: true,
+  versionKey: false,
+  transform(_: any, ret: any) {
+    ret.id = ret._id; // eslint-disable-line no-param-reassign
+    delete ret._id; // eslint-disable-line no-param-reassign
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)